utilities/install_requirements.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+import sys
+
+
+in_venv = getattr(sys, "real_prefix", getattr(sys, "base_prefix", sys.prefix)) != sys.prefix
+pip_call = [sys.executable, "-m", "pip"]
+pip_install = pip_call + ["install"]
+
+if not in_venv:
+    pip_install.append("--user")
+
+subprocess.check_call([*pip_install, "pip", "-U"])
+subprocess.check_call([*pip_call, "uninstall", "depthai", "--yes"])
+subprocess.check_call([*pip_install, "-r", "requirements.txt"])
+try:
+    subprocess.check_call([*pip_install, "-r", "requirements-optional.txt"])
+    if sys.platform == "linux":
+        print(f'$ sudo apt install python3-tk')
+except subprocess.CalledProcessError as ex:
+    print(f"Optional dependencies were not installed (exit code {ex.returncode})")
```

Comment:
Don't uninstall depthai...

Refactor "optional requirements" 
Add a test: first install requirements as is, then (in separate python process, so new libs are sourced), try importing PySimpleGUI (or tkinter). If that fails, display the warn message noting the user to install `sudo apt install python3-tk`

Reply:
@ZigiMigi  pushed commits on his own repo, which I do not have access to. I am not sure how to include his commits into my branch, any ideas?

Reply:
After a bit of tweaking, force pushed @ZigiMigi's commits to this branch. I will fix the installer now.